### DISCUSSION TITLE
为什么我可以执行mpm.execute后可以导出匹配成果文件(csv)，但是无法将结果从函数中输出

### DISCUSSION THREAD

**Discussion by Yixi77** [zdsjjtTLG/TrackIt#11] [Q&A] [answered]
![微信图片_20241220095840](https://github.com/user-attachments/assets/e92d1da5-26b7-4e06-81f3-46d6485fd552)

**Comment by zdsjjtTLG** [comment, marked as answer]
指定了instant_output=True，匹配结果会存储在磁盘，不会在内存中返回
